.github/workflows/dart-metrics-checks.yaml

```diff
@@ -4,18 +4,13 @@ on: [pull_request]
 
 jobs:
   check:
-    name: dart-code-metrics-action
-    runs-on: ubuntu-latest
+    name: code-metrics-action
```

Comment:
Maybe different name?, since it will no longer run code metrics

Reply:
makes sense